src/main/assembly/dist/docs/sword-v1-packaging.html

```diff
@@ -0,0 +1,564 @@
+<!DOCTYPE html>
+<head lang="en">
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title lang="en">Sword Packaging for EASY Sword-v1</title>
+    <style>
+    body {
+        max-width: 1200px;
+        margin: auto;
+        padding: 1em;
+        font-family: "Open Sans",Arial,Helvetica,sans-serif;
+    }
+    article {
+        border: solid 1px;
+        margin-bottom: 1em;
+        margin-right: 1em;
+        border-radius: 0.5em;
+        padding: 1em;
+    }
+    #mapping, #helpIndex {
+        display: inline-block;
+        vertical-align: top;
+    }
+    th {
+        text-align: left;
+    }
+    hr {
+        display: none;
+    }
+    a {
+        text-decoration: none;
+    }
+    a, h1, h2 {
+        color: #248BB8;
+    }
+    a:hover, a:focus {
+        color: #DD291E;
+    }
+    h1, h2, th {
+        font-family: "Glegoo",Georgia,Times New Roman,serif;
+        text-transform: uppercase;
+    }
+    h1, h2 {
+        font-weight: 100;
+    }
+    </style>
+</head>
+<body>
+<h1>Sword Packaging for EASY Sword-v1</h1>
+<p>
+    <a href="http://easy.dans.knaw.nl/">EASY</a> supports a
+    <a href="https://easy.dans.knaw.nl/sword/servicedocument">Sword-v1</a> interface for deposits of
+    datasets. The zip file for a deposit should contain:
+</p>
+<ul>
+    <li>
+        <p>A single file with metadata.</p>
+        <ul>
+            <li>
+                The name of the file should be: <code>DansDatasetMetadata.xml</code>
+            </li>
+            <li>
+                The format is defined by
+                <a href="https://easy.dans.knaw.nl/schemas/md/2016/"
+                >ddm.xsd</a>
+            </li>
+            <li>
+                <a href="https://easy.dans.knaw.nl/schemas/doc/ddm/examples"
+                >Examples</a>
+            </li>
+        </ul>
+    </li>
+    <li>
+        <p>A single folder:</p>
+        <ul>
+            <li>
+                The name of the folder should be: <code>data</code>
+            </li>
+            <li>
+                The full path name of datafiles should not exceed 252 characters.
+            </li>
+            <li>
+                See also:
+                <a href="http://www.dans.knaw.nl/en/deposit/information-about-depositing-data/DANSpreferredformatsUK.pdf"
+                >File formats, preferred formats and accepted formats</a> (pdf)
+            </li>
+        </ul>
+    </li>
+</ul>
+<p>
+    Supplied metadata (DDM) is stored as a file in the dataset.
+    The mapping shows what to expect when downloading the description (EMD) of a published dataset as XML.
+    Elements ignored without a warning may contain elements that will be copied into the EMD.
+    Note that the help articles are written for online deposits,
```

Comment:
The SWORD-deposits are also "online". I think the fact that the help is written for a manual deposit action (as opposed to auto-ingest) is the reason that it refers to GUI-things like "+"-links.